library/src/main/java/ru/terrakok/cicerone/CommandBuffer.java

```diff
@@ -18,7 +21,7 @@
     private Queue<Command[]> pendingCommands = new LinkedList<>();
 
     @Override
-    public void setNavigator(Navigator navigator) {
+    public void setNavigator(@Nullable Navigator navigator) {
```

Comment:
Why `@Nullable`? But for NavigatorHolder.java this method contains `@NotNull`

Reply:
In case of NavigatorHolder i don't see any value in passing null to setNavigator(), cause there's removeNavigator() method. CommandBuffer.setNavigator() parameter should also be notnull, but implementation handles null parameter, so i put nullable annotation here. Also parameter is covariant, so it shouldn't break interface's contract.